### DISCUSSION TITLE
Sidebar missing on version 2022.6.21

### DISCUSSION THREAD

**Discussion by damienrj** [pradyunsg/furo#526] [Potential Issue] [answered]
Hello, on the newest version the sidebar on the right side is now showing up in the browser.  Reverting to 2022.4.7 fixes the issue.

```
Updating furo (2022.4.7 -> 2022.6.21)
```

**Comment by pradyunsg** [comment]
There is not enough detail here to understand what you're seeing.

**Reply by damienrj** [reply]
It is a little harder to describe. 
![image](https://user-images.githubusercontent.com/2729283/190517383-835fefb6-98dd-4442-8e60-1bbcf5f9e293.png)
This image is an example of the contents sidebar that normally gets generated. Our newest builds of the doc page was missing that and looking at the differences in Poetry installs it came down to the update in furo.

**Reply by pradyunsg** [reply]
That's not enough to understand what you're seeing. Could you please share a fuller screenshot and/or a reproducer for the issue?

Mentioning your browser and other details would help as well.

**Comment by damienrj** [comment]
Sure, I will try to make a more self contained example so that I don't need
to mask out content.  The behavior was the same on the current Safri and
Chrome.

**Comment by damienrj** [comment]
This is now longer an issue with the current versions so it looks good now.

**Comment by damienrj** [comment, marked as answer]
I thought it was answered but I have been able to debug it more.

```
# Example
Docs
## Installation and setup
More Docs
# Test
More docs
```
<img width="1140" alt="image" src="https://user-images.githubusercontent.com/2729283/202276274-19142fed-5372-4fe4-8e92-89932c3318de.png">



will work, but if the first heading that isn't the title is # than the side bar doesn't render. 


```
# Example
Docs
# Installation and setup
More Docs
# Test
More docs
```

<img width="1197" alt="image" src="https://user-images.githubusercontent.com/2729283/202275932-5cf78480-5691-4846-badd-925056405b75.png">

```
# Example
With `pip install furo==2022.4.7`
# Installation and setup
More Docs
# Test
More docs
```
If I downgrade furo than both examples work
<img width="1177" alt="image" src="https://user-images.githubusercontent.com/2729283/202276784-474fd9eb-4ffe-4de0-b202-3fd7ae79c1ba.png">

```
# Example
With `pip install furo==2022.4.7`
## Installation and setup
More Docs
# Test
More docs
```
<img width="1144" alt="image" src="https://user-images.githubusercontent.com/2729283/202276940-75da8eb4-ded1-44cc-b34c-96d97940c9e6.png">

**Reply by pradyunsg** [reply]
> ```
> # Example
> Docs
> # Installation and setup
> More Docs
> # Test
> More docs
> ```

Your heading structure is "wrong" (well, as far as Furo and Sphinx are concerned), which is why nothing is being shown. There should only be one `h1` on the page, and everything else should be nested under it. The following will do the right thing:

```md
# Example
Docs
## Installation and setup
More Docs
## Test
More docs
```

See https://developer.mozilla.org/en-US/docs/Web/HTML/Element/Heading_Elements#avoid_using_multiple_h1_elements_on_one_page for the details.

**Reply by pradyunsg** [reply]
If you did that to get those headings into the left sidebar, that's actually not a supported thing: Furo is designed (intentionally) to separate the site structure (i.e. left sidebar) from the page content (i.e. right sidebar).

**Reply by damienrj** [reply]
Yeah, that makes sense.   We have not been using it for the left sidebar. I think it was less obvious before since we were using the 


```
title
=====
```

And since the older version of the package was still rendering the incorrect version we didn't know so we can update our docs. Thanks
